docs/en/connector-v2/source/SftpFile.md

```diff
@@ -176,6 +195,13 @@ The compress codec of files and the details that supported as the following show
 - orc/parquet:  
   automatically recognizes the compression type, no additional settings required.
 
+### archive_compress_codec [string]
+
+The compress codec of archive files and the details that supported as the following shown:
+
+- zip: text,json,xml
+- tar: text,json,xml
+
```

Comment:
@corgy-w Can tar.gz and more common compression types be supported?

Reply:
get

Reply:
zip, tar, and tar.gz are temporarily supported